alws/app.py

```diff
@@ -37,5 +38,5 @@ async def shutdown():
 
 
 for module in (builds, build_node, distro, platforms, projects, releases,
-               tests, users):
+               repositories, tests, users):
```

Comment:
Why did you not move it to some external constant which will be imported and used here?